### ISSUE TITLE
ev3 Color sensor suddenly shuts down

### ISSUE THREAD

**Issue by Abdelrhman-Mazen** [a10036gt/EV3-ColorAdvance-Block#2]
I have used this block in WRO 2022 it caused the sensor to what seemed to me randomly shut down 2 teams were using it one with a gyro sensor and hi technic and the other team with 2 other color sensors and a gyro. I don't know the solution but it started when using this block, and then later in 2023 a line tracking team using 4 color sensors without the block in their code .  I don't know if it is the same sensor,hub,both or neither .also a teammate thought that he saw the solution on github but we couldn't find it .but if someone has anything to help or insights

**Comment by a10036gt** [comment]
The EV3 Color Sensor randomly shut down issue is famous problem, In my test use old EV3 brick (made before 2016) will solve the problem, some teams say turn off the BlueTooth the problem might be solve.

**Comment by Abdelrhman-Mazen** [comment]
[a10036gt](https://github.com/a10036gt) i don't really understand are you saying the solution is using a brick that is made before 2016? or I am understanding something wrong

**Comment by a10036gt** [comment]
Yes, In our test Old EV3 did't have this problem, We don't know why but it actually work. Some teams say turn off the BlueTooth the problem might be solve.

And due to Color Sensor firmware issue, we guest the randomly shut down is sensor watchdog has been trigger.
